src/locales/en/translation.json

```diff
@@ -92,6 +93,12 @@
     "valuePair": "{{value, number}}"
   },
   "contract": {
+    "code": "Code",
+    "copyButton": "Copy {{subject}}",
+    "creationByteCode": "Creation ByteCode",
```

Comment:
mocks refer to this as `Deployed ByteCode`. no idea which is valid.

Reply:
Asked Don. WIll update if necessary.